src/AppInstallerRepositoryCore/ArpVersionValidation.cpp

```diff
@@ -67,11 +67,12 @@ namespace AppInstaller::Repository
             {
                 if (manifestArpVersionRange.Overlaps(arpInIndex))
                 {
-                    std::string errorMsg = Manifest::ManifestError::ArpVersionOverlapWithIndex;
-                    errorMsg.append("[" + arpInIndex.GetMinVersion().ToString() + ", " + arpInIndex.GetMaxVersion().ToString() + "]");
-                    AICLI_LOG(Repo, Error, << errorMsg);
+                    std::string context = (" [" + arpInIndex.GetMinVersion().ToString() + ", " + arpInIndex.GetMaxVersion().ToString() + "]");
+                    auto validationError = Manifest::ValidationError(Manifest::ManifestError::ArpVersionOverlapWithIndex, context);
+                    
+                    AICLI_LOG(Repo, Error, << validationError.GetErrorMessage());

```

Comment:
>AICLI_LOG(Repo, Error, << validationError.GetErrorMessage()); [](http://example.com/codeflow?start=20&length=61)

Please also log the context, this is the important part for log viewer

Reply:
The context is part of the error message, previous implementation does log the message with the assumption, i don't see why it should be any different here.

Reply:
Previous logged message is "Arp Version Overlap [1.0, 2.0]", the range is part of the message
With the new code, the failing range is in the context part of the ValidationError, new code using GetErrorMessage() only logs "Arp Version Overlap" without the violating version range, which is less helpful.